docs/README.rst

```diff
@@ -122,6 +165,7 @@ If you configure the state to download the formulas from repositories that
 you control, then you can safely enable the
 ``salt_formulas:git_opts:default:update`` pillar setting to ``True``.
 
+
 ``Configuration``
 =================
 Every option available in the templates can be set in pillar. Settings under 'salt' will be overridden by more specific settings under ``salt['master']``, ``salt['minion']`` or ``salt['cloud']``. Options specified in ``salt['minion']`` which are not present in the default configuration file will be added to the end of the configuration file.
```

Comment:
Not the right line for this comment but we also add the `Testing` section from `template-formula`, right at the end of the `README`.